dataactbroker/README.md

```diff
@@ -932,7 +932,9 @@ This endpoint lists submissions for all agencies for which the current user is a
     "order": "desc",
     "d2_submission": False,
     "filters": {
-        "submission_ids": [123, 456]
+        "submission_ids": [123, 456],
+        "start_date": "01/01/2018",
```

Comment:
To answer your question, I think these start/end dates might get in the way if we end up wanting to be able to filter by two different types of date ranges. I also think an array with two values wouldn't be the best way to do it either. This is what I recommend:
```
"filters": {
        "submission_ids": [123, 456],
        "last_modified_range": {
            "start_date": "01/01/2018",
            "end_date": "01/10/2018"
        }
}
```